src/components/data/LocalVariablesContent.js

```diff
@@ -47,25 +48,57 @@ const LocalVariablesContent = ({ workspace, workspace: { workspace: { googleProj
     setEditIndex()
     setEditKey()
     setEditValue()
+    setEditDescription()
     setEditType()
   }
 
+  const DESCRIPTION_MAX_LENGTH = 200
+  const DESCRIPTION_SUFFIX = '__DESCRIPTION__'
+
+  const isDescriptionKey = k => k.endsWith(DESCRIPTION_SUFFIX)
+
+  const toDescriptionKey = k => k + DESCRIPTION_SUFFIX
+
+  const renameAttribute = attr => {
+    const [k, v] = attr
+    return isDescriptionKey(k) ?
+      {
+        key: k.substring(0, k.length - DESCRIPTION_SUFFIX.length),
+        description: v
+      } : {
+        key: k,
+        value: v
+      }
+  }
+
+  const getDisplayedAttribute = arr => {
```

Comment:
This function might be a good candidate to write a unit test for. See the unit test files in [terra-ui/src/_tests/](https://github.com/DataBiosphere/terra-ui/tree/dev/src/_tests). Use `yarn test` in the _tests directory to run them.

Reply:
Done - wrote unit tests for this function and renameAttribute.

Reply:
Also fixed the error you encountered.